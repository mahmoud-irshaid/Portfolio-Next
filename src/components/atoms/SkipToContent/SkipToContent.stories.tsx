/* eslint-disable import/no-anonymous-default-export */
import { SkipToContent } from './SkipToContent';

export default {
	title: 'atoms/SkipToContent',
	component: SkipToContent,
};

export const SkipToContentStory = () => <SkipToContent />;

SkipToContentStory.storyName = 'SkipToContent';
