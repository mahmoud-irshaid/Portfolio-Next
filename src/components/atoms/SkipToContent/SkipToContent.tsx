const SkipToContent = () => {
	return (
		<a
			href="#content"
			className="sr-only left-2 top-2 focus:not-sr-only focus:absolute focus:z-[1000]"
		>
			Skip to content
		</a>
	);
};

export { SkipToContent };
