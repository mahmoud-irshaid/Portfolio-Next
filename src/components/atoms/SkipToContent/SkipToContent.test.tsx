import { SkipToContent } from '../SkipToContent';

import { render } from '@testing-library/react';

describe('SkipToContent', () => {
	it('renders correctly', () => {
		const { container } = render(<SkipToContent />);
		expect(container).toMatchSnapshot();
	});
});
