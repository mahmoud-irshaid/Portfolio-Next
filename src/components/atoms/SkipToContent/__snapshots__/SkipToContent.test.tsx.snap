// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SkipToContent renders correctly 1`] = `
<div>
  <a
    class="sr-only left-2 top-2 focus:not-sr-only focus:absolute focus:z-[1000]"
    href="#content"
  >
    Skip to content
  </a>
</div>
`;
